src/pools/thread/dynamic.ts

```diff
@@ -5,10 +5,10 @@ import type { ThreadWorkerWithMessageChannel } from './fixed'
 import { FixedThreadPool } from './fixed'
 
 /**
- * A thread pool with a dynamic number of threads, but a guaranteed minimum number of threads.
+ * A worker pool with a dynamic number of workers, but a guaranteed minimum number of workers.
```

Comment:
Do we need to change this file? It is specifically the DynamicThreadPool :thinking:

Reply:
All other JSDoc talk about worker, including the one for FixedThreadPool, not thread except that one.

Reply:
Not really true, e.g.:

https://github.com/pioardi/poolifier/blob/f416c0987c6fe2d452de7e854b0d20f094448dd4/src/pools/thread/fixed.ts#L16